testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,24 @@
+from MDAnalysis.coordinates.GRO import GROReader, GROWriter
+from MDAnalysisTests.coordinates.base import BaseReference, BaseReaderTest
+from MDAnalysisTests.datafiles import COORDINATES_GRO
+import numpy as np
+
+
+class GROReference(BaseReference):
```

Comment:
Yes, I just wanted to keep it separate while I'm working on it so it don't get it mixed. Once I'm done I'll move it to its correct place.